Dokan/Dokan/Scenes/ProductDetails/TitleQuantity/ProductTitleQuantityView.swift

```diff
@@ -65,13 +111,11 @@ class ProductTitleQuantityView: UIView {
     }
 }
 
-extension ProductTitleQuantityView {
-    struct ViewModel {
-        let title: String
-        let currency: String
-        let price: String
-        let reviewAverage: Double
-        let reviewCount: Int
-        let stockCount: Int
-    }
+public struct tQViewModel {
```

Comment:
- Let's use a nested view model to keep the convention. 
- Please make sure to start a type name with a capital case!